examples/add_labels_with_features.py

```diff
@@ -56,7 +58,7 @@
     label_image,
     name='segmentation',
     features=label_features,
-    color=color,
+    colormap=DirectLabelColormap(color_dict=colors),
```

Comment:
Yes. The old behavior was adding transparent color for the background. There is a need to add `0: 'transparent'` to color dict.

Reply:
I think we should maybe add the background = 0 behaviour as default to the DirectLabelColormap to preserve the old behaviour (and this is the most intuitive behaviour for our users, I think). Then it could be disabled by passing `background_value=None` to the constructor. Thoughts?

Reply:
I do not like it. It complicates API. But we could to this with `direct_colormap` helper function https://github.com/napari/napari/blob/3c58ab41e1a1772f133852a6c6c4adf7e209a576/napari/utils/colormaps/colormap_utils.py#L567-L585

Reply:
Ah, I see that the background value *is* 0 by default. Well, I think by default the background should always be fully transparent — that is, if there is not a key for the background value, we add it. @Czaki?

Reply:
I've implemented this in ddbe9c8 (with a slight tweak to the docstring in 8709db7)